vnext/ReactUWP/Modules/I18nModule.h

```diff
@@ -10,19 +10,42 @@
 namespace react {
 namespace uwp {
 
+class I18nHelper;
+
 class I18nModule final : public react::windows::II18nModule {
  public:
   using I18nInfo = std::pair<std::string, bool>;
   static I18nInfo GetI18nInfo(); // Must be called from a UI thread
 
   // II18nModule
-  I18nModule(I18nInfo &&i18nInfo);
+  I18nModule();
 
   std::string getLocaleIdentifier() override;
   bool getIsRTL() override;
+  void setAllowRTL(bool allowRTL) override;
+  void setForceRTL(bool forceRTL) override;
 
  private:
-  I18nInfo m_i18nInfo;
+  I18nHelper &m_helper;
 };
+
+class I18nHelper {
+ public:
+  static I18nHelper &Instance();
```

Comment:
>) [](start = 30, length = 1)

Please consider to add noexcept to methods and constructors to keep binary code smaller.